_layouts/default.md

```diff
@@ -67,9 +67,10 @@
 <hr />
 <div>
   <h4 class="text-center text-muted">powered by <img src="{{ site.baseurl }}/app/logo.png" class="mr-2" style="height: 1.2em;" /></h4>
-  <p class="text-center text-muted mb-0" style="font-size: 0.8em">Copyright © Form.io LLC 2019. All rights reserved</p>
+  <p class="text-center text-muted mb-0" style="font-size: 0.8em">Copyright © Form.io LLC 2020. All rights reserved</p>
 </div>
 <p class="text-center text-muted mb-0" style="font-size: 0.8em">Renderer v<span id="renderer-version"></span></p>
+<script src="{{ site.baseurl }}/dist/formio.full.js"></script>
```

Comment:
This is already included on line 32.  Why are we doing it again here?

Reply:
If we want to have an access to Formio on DOM we should use scripts on the body. On the line 32 scripts includes on like a setting on the head. But we are using Formio.version also and when we're trying to include the embed file it's breaking global variable Formio because of an unknown reason. I couldn't find out what is the reason but I know setting scripts for DOM is better to include it on the body.

Reply:
This is probably because the "embed" overrides the "instance" that is already defined. I think the solution here is to add an if statement to the pages so that they will not include the formio.form.js at the beginning and the "embed" page would set this flag since it includes it in the embed.